api/getPageContentByPathname/index.spec.ts

```diff
@@ -1,35 +1,51 @@
 import getPageContentByPathname from './'
-import { CONTENTFUL_API_URL } from '../constants'
-import { rest } from 'msw'
+import { GET_PAGE_CONTENT_BY_PATHNAME } from './constants'
+import { graphql } from 'msw'
 import { setupServer } from 'msw/node'
 import 'whatwg-fetch'
 import type { PageContentProps } from '../../components/Main/PageContent'
 
-const mockPageContent: PageContentProps = {
-  content: 'mock content',
-  title: 'mock title',
-}
+it('should get page content', async () => {
```

Comment:
Let's use `test` rather than `it` and not bother repeating `should` every time.

What exactly are we testing here? What is `getPageContentByPathname` actually doing and what do we gain by testing that?

Reply:
OK. I updated the test function name and the test title according to what should really get by the function. 
`getPageContentByPathname` should load page content from CMS. The first test is to test the function when an array with a page content element returns.  And the second test is to test the function when an empty array returns.

Reply:
My point is, it appears that all this test is testing is the _mock_. And that it destructures the returned data. Hardly seems worth it. Might be better as an integrated test at the page level? I'll have to think about it. What do you think, @reubenberghan ?

Reply:
If we really want to test the logic here we could abstract this into a pure function that takes the raw data and returns the expected shape and then unit test this. But in this case where we're just reaching down a few levels in the returned object and returning that raw data probably not necessary.